platform-sdk/swirlds-logging/src/test/java/com/swirlds/logging/api/internal/format/EpochFormatUtilsTest.java

```diff
@@ -19,10 +19,12 @@
 import static java.time.ZoneOffset.UTC;
 import static org.assertj.core.api.Assertions.assertThat;
 
+import com.swirlds.base.test.fixtures.io.WithSystemError;
 import java.time.Instant;
 import java.time.format.DateTimeFormatter;
 import org.junit.jupiter.api.Test;
 
+@WithSystemError
```

Comment:
Why? Does the EpochFormatUtils create sys out? Or is that the emergency logger that is used in the util.

Reply:
Yes the utils logs to the emergency logger when it can't parse the date.